contracts/test/unit/EigenDABlobUtils.t.sol

```diff
@@ -59,6 +59,7 @@ contract EigenDABlobUtilsUnit is BLSMockAVSDeployer {
                     abi.encodeWithSelector(
                         EigenDAServiceManager.initialize.selector,
                         pauserRegistry,
+                        0,
```

Comment:
just to confirm, it means unpaused right?

Reply:
yes unpaused